upgradeability_using_fixed_storage/contracts/OwnedUpgradeabilityStorage.sol

```diff
@@ -0,0 +1,37 @@
+pragma solidity ^0.4.21;
+
+
+/**
+ * @title OwnedUpgradeabilityStorage
+ * @dev This contract keeps track of the upgradeability owner
+ */
+contract OwnedUpgradeabilityStorage {
+  // Storage position of the owner of the contract
+  bytes32 private constant upgradeabilityOwnerPosition = keccak256("zeppelin_os.proxy.owner");
```

Comment:
I know I originally suggested this name, but maybe change it to `org.zeppelinos.proxy.owner`, to go with a more java-like approach?